code/API_definitions/geofencing-subscriptions.yaml

```diff
@@ -362,11 +361,9 @@ components:
           $ref: "#/components/schemas/Protocol"
         sink:
           type: string
-          format: url
+          format: uri
           description: The address to which events shall be delivered using the selected protocol.
           example: "https://endpoint.example.com/sink"
-        sinkCredential:
```

Comment:
SinkCredential is to be removed from the notification event, but not from the subscription request:
https://github.com/camaraproject/Commonalities/blob/main/artifacts/camara-cloudevents/event-subscription-template.yaml#L282